loopback.go

```diff
@@ -480,11 +531,150 @@ var _ fs.NodeRenamer = (*Node)(nil)
 // Rename implements fs.NodeRenamer interface for *Node
 func (n *Node) Rename(ctx context.Context,
 	req *fuse.RenameRequest, newDir fs.Node) (err error) {
+	time.Sleep(latency)
 	np := filepath.Join(newDir.(*Node).realPath, req.NewName)
 	op := filepath.Join(n.realPath, req.OldName)
 	defer func() {
 		log.Printf("%s.Rename(%s->%s): error=%v",
 			n.realPath, op, np, err)
 	}()
+	defer func() {
+		if err == nil {
+			n.fs.moveAllxattrs(ctx, op, np)
+		}
+	}()
 	return os.Rename(op, np)
 }
+
+func xattrNameMutate(name string) string {
```

Comment:
OK; I'll try to find out and add a comment, or at least explaining that's what osxfuse does